tests/testapp/test_helpers.py

```diff
@@ -15,35 +15,37 @@ def __new__(cls, id_):
         return super(WithId, cls).__new__(cls, id_, 'type name')
 
 
-def run_testcase_with_django_runner(
```

Comment:
@belugame what was the reason for using a class instead of the helper method? As for internal refactoring, I'm happy for it, but from the usage perspective, now there is more duplication inside the tests, and more lines of code in each test method. I would have preferred to keep the helper method, and that internally would construct the class object and call run on it - that way, the tests don't have to change should it be refactored again